README.md

```diff
@@ -11,20 +11,55 @@ The purpose of the system is to enable analysts to quickly review monitored gove
 For more, see [the Version Tracking page on the EDGI website](https://envirodatagov.org/version-tracking/) and watch this
 [50-minute Analyst training video](https://www.dropbox.com/s/ciixvu612ktf4nt/new_tracking_training.mp4?dl=0).
 
-## Specification
+## How to Help
 
-In order to be useful, tools will need to be able to do the following:
+Start by reading the Project Overview below and thinking about which aspects of
+the projects spark your interest.
 
-1. read archive directories and identify comparison candidates for a given URL
-2. perform diffs on large numbers of pages (a useful scale for our work is the "subdomain" -- a full or partial government domain that we have identified as relevant to environment/climate change)
-3. filter those diffs to eliminate irrelevant, trivial, or repetitive changes
-4. output results in a form that is immediately acessible/helpful to an analyst. One likely format is a CSV file containing at least the following information:
-   - page title
-   - page url
-   - comparison dates
-   - **link to user-friendly diff** -- this can be local or served over http
+We expect to deploy a minimal operational version of this application by the
+end of March. Starting from that baseline, we will post many feature requests,
+bug reports, and documentation needs.
 
-## Architecture
+See also the [contributor guidelines](https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md).
+
+## Project Overview
+
+### Specification
+
+1. Access captured data (starting with HTML, later encompassing more types) from
+   multiple archival sources including Versionista, PageFreezer, and the
+   Internet Archive.
+2. Compare version of the same page over time --- potentially using multiple
+   different strategies.
+3. Automatically filter out "uninteresting" or repetitive changes: for example,
+   the "Page Last Viewed" timestamp updated or the same news article was added
+   to 100 pages from the same website.
+4. Prioritize the changes most likely to be "interesting," meaning that some
+   item of importance to fact-based governance was deleted or changed in a
+   harmful way.
+5. Present changes to human analysts with useful visualizations and statistics
+   to help them differentiate interesting changes. Each user will have been
+   assigned a "subdomain", a full or partial government domain that has been
+   identified as relevant to fact-based governance.
+6. Collect annotations from the analysts. Use this to flag and draw broader
```

Comment:
I think something may need to be sketched out about "public attention"? How will this data be taken and worked with by analysts communicating with journalists, building reports, or just viewed by the public?